### PR TITLE
Update library dependencies and TwoPaneLayout navigation methods

### PR DESCRIPTION
**Pull request by khalp**
## 📃 Description

Updates dependencies for all libraries, and makes the following changes to TwoPaneLayoutNav:
- removes incorrect support for `launchSingleTop` nav option
- adds `navigateUpTo` method to `TwoPaneNavScope`

## 🤔 Motivation and Context

Keeps libraries up to date and makes TwoPaneLayoutNav usage more flexible

## 🧪 How Has This Been Tested?

Ran existing tests and added new tests for `navigateUpTo`

## ⚒️ Does the PR contain new tests or refactored old ones?

Both

## 📦 Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

**Review by bimiron — APPROVED** [review]
Looks good!

**Comment by AlinArgeanu** [comment]
LGTM, nice job!

**Review by AlinArgeanu — APPROVED** [review]

**Review by andreidiaconu — APPROVED** [review]
LGTM

File: TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneScope.kt [hunk 1]
```diff
@@ -64,6 +64,17 @@ interface TwoPaneNavScope {
...
 
+    /**
+     * Navigates up to the given destination and pops any intermediate destinations off the backstack.
+     *
+     * @param route: route of the destination to pop up to
+     * @param inclusive: if true, the given destination will also be popped off the backstack
+     */
+    fun NavHostController.navigateUpTo(
+        route: String,
+        inclusive: Boolean = true
```
[the remark this answers is not in the document]
Comment (khalp) [reply TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneScope.kt]: Great point, I'll change that!

**Pull request merged**